mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLHostOps.td

```diff
@@ -228,4 +228,30 @@ def SYCLHostScheduleKernel : SYCL_HostOp<"schedule_kernel",
   let hasVerifier = 1;
 }
 
+def SYCLHostSubmitOp : SYCL_HostOp<"submit",
+    [DeclareOpInterfaceMethods<SymbolUserOpInterface>]> {
+  let summary = "Submits a SYCL kernel.";
+  let description = [{
+    This operation represents the submission of a SYCL kernel
+    (`sycl::queue::submit`). In addition to the function implementing the CGF,
+    this operation receives the values captured (by reference) by this.
+
+    Example:
+    ```
+    sycl.host.submit %queue([%buff_a, %buff_b]@kernels::@k0) -> %event
```

Comment:
Nit: `@kernels::@k0` is a bit misleading here, as we've used this as an example when talking about actual kernel functions (i.e. `gpu.func` with `kernel` attribute).

Reply:
True, I changed the definition also so that the reference is flat (should be in the same module)